api/src/main/java/uk/gov/hmcts/payment/api/dto/TelephonyPaymentsReportDto.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.payment.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.hmcts.payment.api.model.PaymentStatus;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+@Builder(builderMethodName = "telephonyPaymentsReportDtoWith")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class TelephonyPaymentsReportDto {
+    @JsonProperty("Resp_Service Name")
```

Comment:
I'm double checking this with Ali - just doesn't look right unless third party tools need the headings fixed - it's a new report though.

Reply:
Agreed, this is what he wanted, I suggest that all reports get standardised